mriqc/reports/individual.py

```diff
@@ -23,8 +23,10 @@
 """Encapsulates report generation functions."""
 from pathlib import Path
 from json import loads
-from pkg_resources import resource_filename as pkgrf
 from nireports.assembler.report import Report
+from niworkflows.data import Loader
+
+_load_data = Loader(__package__)
```

Comment:
I don't think this is going to work quite how you want. I'm pretty sure this will load relative to `mriqc.reports.individual`. I would suggest instead having `data/__init__.py` define `load_data` and import from there.

Reply:
would `_load_data = Loader("mriqc")` work?

Reply:
Yes.